addons/crm/models/crm_team.py

```diff
@@ -149,14 +149,22 @@ def action_your_pipeline(self):
         tree_view_id = self.env.ref('crm.crm_case_tree_view_oppor').id
         form_view_id = self.env.ref('crm.crm_case_form_view_oppor').id
         kanb_view_id = self.env.ref('crm.crm_case_kanban_view_leads').id
-        action['views'] = [
-                [kanb_view_id, 'kanban'],
-                [tree_view_id, 'tree'],
-                [form_view_id, 'form'],
-                [False, 'graph'],
-                [False, 'calendar'],
-                [False, 'pivot']
-            ]
+
+        mode_string = action.get('view_mode', '')
+        view_mode = mode_string and mode_string.split(',') or ['kanban', 'tree', 'form', 'graph', 'calendar', 'pivot']
```

Comment:
Just to be nitpicking because it's cleaner:
```python
view_mode = mode_string.split(',') if mode_string else ['kanban', 'tree', 'form', 'graph', 'calendar', 'pivot']
```